molecule/default/converge.yml

```diff
@@ -0,0 +1,64 @@
+---
+- name: Converge
+  hosts: all
+  become: true
+
+  vars:
+    drupal_log_path: /var/log/vhosts/account/project/drupal.log
+    drupal_syslog_read_users:
+      - johndoe
+
+  pre_tasks:
+
+    - name: Update apt cache.
+      apt: update_cache=true cache_valid_time=600
+      changed_when: false
+      when: ansible_os_family == 'Debian'
+
+    - name: Creates directory
+      file:
+        path: /var/log/vhosts/account/project
+        state: directory
+
+    - name: Creates directory
+      file:
+        path: /etc/rsyslog.d
+        state: directory
+
+    - stat: path=/var/log/vhosts/account/project/drupal.log
+      register: dlog
+
+    - name: Ansible create file if it doesn't exist
+      file:
+        path: "/var/log/vhosts/account/project/drupal.log"
+        state: touch
+      when: dlog.stat.exists is defined and not dlog.stat.exists
```

Comment:
Creating the containing directory for the logfile is fine, but be careful not to duplicate [stuff that's already happening in the role](https://github.com/AcroMedia/ansible-role-drupal-syslog/blob/master/tasks/main.yml#L7). This task breaks our ability to test the role because the file gets created with wrong permissions, which means syslog won't be able to write to the file.

The rest of this pre_tasks section is spot on, except i think we should start moving "setup" conditions like this over to `prepare.yml`, which is what that file is designed for.